### ISSUE TITLE
Unusual behavior of flex objects

### ISSUE THREAD

**Issue by xkiixkii** [google-deepmind/mujoco#1900]
Hi,

I am simulating the free fall of an insole, but instead of dropping straight down as we would intuitively expect, it rotates as it falls. 
And I noticed that the plane geometry is not frictionless by default, I would like to ask why it continues to move after hitting the ground.

Here is a model which explains my question:

<details>
  <summary>insole.xml</summary>

```XML
<mujoco>
	<include file="floor.xml"/>
	<compiler meshdir="meshes/" />
	<extension>
		<plugin plugin="mujoco.elasticity.solid" />
	</extension>
	<worldbody>
		<body name="insole_fixed" pos=".12 -.05 .456">
			<flexcomp name="insole" mass=".5" dim="3" type="gmsh" file="insole.msh" rgba=".1 .9 .1 1">
				<edge equality="true" />
				<plugin plugin="mujoco.elasticity.solid">
					<config key="young" value="5e4" />
					<config key="poisson" value="0.1" />
					<config key="damping" value="1e-4" />
				</plugin>
			</flexcomp>
		</body>
	</worldbody>
</mujoco>

```

</details>

<details>
  <summary>floor.xml</summary>

```XML
<mujoco>
	<option timestep="0.005"/>
	<visual>
		<rgba haze="0.15 0.25 0.35 1" />
		<quality shadowsize="8192" />
		<global azimuth="220" elevation="-30" />
	</visual>
	<asset>
		<texture type="skybox" builtin="gradient" rgb1="0.3 0.5 0.7" rgb2="0 0 0" width="512" height="3072" />
		<texture type="2d" name="groundplane" builtin="checker" mark="edge" rgb1="0.2 0.3 0.4" rgb2="0.1 0.2 0.3" markrgb="0.8 0.8 0.8" width="300" height="300" />
		<material name="groundplane" texture="groundplane" texuniform="true" texrepeat="5 5" reflectance="0.2" />
	</asset>
	<worldbody>
		<light pos="0.3 0 1.5" dir="0 0 -1" directional="true" />
		<geom name="floor" pos="0 0 0" size="0 0 .05" type="plane" material="groundplane" />
	</worldbody>
</mujoco>


```

</details>
Here is a screenshot, illustrating my question:

![Peek 2024-08-13 15-00](https://github.com/user-attachments/assets/77c5710e-e92a-42db-bcba-2138afff058a)

Here is the mesh that I use
[insole.zip](https://github.com/user-attachments/files/16599981/insole.zip)

**Comment by quagla** [comment]
Have you tried reducing the radius? 

https://mujoco.readthedocs.io/en/latest/XMLreference.html#body-flexcomp-radius

**Comment by yuvaltassa** [comment]
@quagla radius should only affect internal forces, so not clear to me that this would make a difference. Even if it does (OP, let us know!), this seems like a bug since internal forces should not affect global angular momentum...

**Comment by xkiixkii** [comment]
@quagla Thank you for your reply,
I have tried to reduce the radius to 0.001 and did some ajustment of young's module, now it can free fall normaly.
![normal](https://github.com/user-attachments/assets/cfe33ab1-ee7c-4021-a7aa-5bbcd64cd7ca)

@yuvaltassa
It seems like the internal forces do affect the global angular momentum.
As I observed, if the radius of flex elements is large enough to produce a contact force between them, all of the internal contact forces will be applied on the body surface, even though it hasn't made contact with anything else.
![anormal](https://github.com/user-attachments/assets/bf05e2b4-7ce3-4f7b-95ad-b0daaa85b769)
